pyproject.toml

```diff
@@ -43,7 +43,26 @@ Documentation = "https://more-itertools.readthedocs.io/en/stable/"
 [tool.flit.module]
 name = "more_itertools"
 
-[tool.black]
-line-length = 79
-target-version = ["py39"]
-skip-string-normalization = true
+[project.optional-dependencies]
+lint = [
+    "ruff >= 0.8",
+]
+
+[tool.ruff]
+indent-width = 4
+line-length = 80
+target-version = "py39"
+
+[tool.ruff.lint]
+select = ["B", "E", "F", "I", "UP"]
+ignore = ["B904", "B028", "F841", "E741"]
```

Comment:
Could you describe why these in particular are ignored?

Reply:
I found the [setup.cfg](https://github.com/more-itertools/more-itertools/blob/master/setup.cfg), but I find no explanation why the configuration in this PR differs. I need to sleep, add the rules from [this line 9](https://github.com/more-itertools/more-itertools/blob/master/setup.cfg#L9) + any suggestions are welcomed.

Thank you and sorry for inconveniences.